code/config/resCodes/user/profile.js

```diff
@@ -0,0 +1,30 @@
+resCode = {
+  report:{
+      success:{
+        status:200
+      },
+      userNotFound:{
+        status:404,
+        message:'Could not find user by ID'
+      },
+      userNotGiven:{
+        status:400,
+        message:'No user id was given'
+      }
+  },
+  profileReport:{
+      success:{
+        status:200
+      },
+      profileNotFound:{
+        status:404,
+        message:'Could not find user profile'
+      },
+      tripcountNotFound:{
+        status:400,
+        message:'Could not find user trip count'
+      }
+    }
+}
```

Comment:
The rescodes should be formatted so that the inside object corresponds to the name of the api call. For example, since right now there is only one profile api report and profileReport should be changed to one json object called profile since the api call is called profile. Let me know if this doesn't make sense.

Reply:
If I understand this correctly I should modify "profileReport" to "profile" so that it matches the api name?

Reply:
Ya, so combining the two of these into one called profile. Let me know if this still isn't clear

Reply:
Ok. That's done!